src/render.py

```diff
@@ -165,6 +238,13 @@ def render_post(post: Tag, image_map: ImageMap) -> RenderedPost:
         [x for x in [character, screen_name, author] if x is not None]
     )
 
+    for inline_img in content.find_all("img"):
+        mapped_image = image_map.get_image_name(inline_img["src"])
+        if mapped_image is not None:
+            inline_img["src"] = "../%s" % mapped_image
+        else:
+            inline_img["src"] = "data:,"
```

Comment:
It's handled as an empty text file; in browsers, and in both readers to which I have convenient access, this causes it to be treated as a broken image, with its alt text displayed if nonempty. This is mildly ugly, but it's the best solution I'm aware of to the "image failed to download" case, since external linking of images is non-standards-compliant.